atmosphere.yaml

```diff
@@ -36,3 +36,11 @@ sources:
           same_names: True
       xarray_kwargs:
         use_cftime: True
+  CRU_TS:
+    description: Climatic Research Unit (CRU) Time-Series (TS) of high-resolution gridded data of month-by-month variation in climate
+    metadata:
+      url: https://crudata.uea.ac.uk/cru/data/hrg
+      comment: CEDA archive requires authorization, for setting up a certificate, please follow instructions from https://help.ceda.ac.uk/article/4712-reading-netcdf-with-python-opendap
+    args:
+      path: "{{CATALOG_DIR}}/atmosphere/cru_ts.yaml"
+    driver: intake.catalog.local.YAMLFileCatalog
```

Comment:
for now this ok. in future a solution without the need of a nested catalog would be great.

Reply:
yes, that would be great! For now, i went with the [pangeo](https://github.com/pangeo-data/pangeo-datastore/tree/master/intake-catalogs) solution, they have nested catalogs for the `ocean`.